<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwindow</class>
 <widget class="QWidget" name="mainwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mainwindow</string>
  </property>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_conn">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>30</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="sp_l">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="sp_r">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>250</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>380</y>
     <width>188</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QDoubleSpinBox" name="sp_la"/>
    </item>
    <item row="1" column="2">
     <widget class="QDoubleSpinBox" name="sp_ld"/>
    </item>
    <item row="0" column="1">
     <widget class="QDoubleSpinBox" name="sp_lw"/>
    </item>
    <item row="2" column="1">
     <widget class="QDoubleSpinBox" name="sp_ls"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>380</y>
     <width>236</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="1">
     <widget class="QDoubleSpinBox" name="sp_rw">
      <property name="decimals">
       <number>6</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QDoubleSpinBox" name="sp_ra"/>
    </item>
    <item row="1" column="2">
     <widget class="QDoubleSpinBox" name="sp_rd"/>
    </item>
    <item row="2" column="1">
     <widget class="QDoubleSpinBox" name="sp_rs"/>
    </item>
   </layout>
  </widget>
  <widget class="QSlider" name="sensor">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>310</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
